### PR TITLE
Benchmark

### PR DESCRIPTION
**Pull request by benoit-canet**
most comments from Amos where addressed except one about the 
cluster where I didn't find the associated code in github and the portability
to gce and libvirt that Shlomi suggest to implement later.

### CODE REVIEW

File: sdcm/tester.py [hunk 1]
```diff
@@ -163,6 +163,7 @@ def setUp(self):
         db_node_address = self.db_cluster.nodes[0].private_ip_address
         self.loaders.wait_for_init(db_node_address=db_node_address)
         nodes_monitored = [node.public_ip_address for node in self.db_cluster.nodes]
+        nodes_monitored += [node.public_ip_address for node in self.loaders.nodes]
```
Comment (lmr) [inline sdcm/tester.py]: Interesting, I thought we only had provisions to monitor the database nodes. If we can also monitor loaders, that is cool.

**Comment by kongove** [comment]
@benoit-canet Sorry, the job failed after merged your PR. The 'name' isn't defined in GCE, Libvirt backend.

**Comment by kongove** [comment]
@benoit-canet  ping

**Comment by benoit-canet** [comment]
@amoskong pong
Sorry I loose track of this. I will rebase it and iterate.

**Comment by roydahan** [comment]
What are we doing with this PR?

**Pull request closed without merging**